src/BenchmarkManager.py

```diff
@@ -126,43 +209,82 @@ def run_benchmark(self, benchmark_backlog: list, repetitions: int):
         git_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", )
         git_revision_number, git_uncommitted_changes = get_git_revision(git_dir)
 
+        job_status_count_total = {}
         try:
```

Comment:
Since PR #112 was accepted, the KeyboardInterrupt is handled slightly differently (with a break_flag instead of raising an exception). I think this new mechanism facilitates the option to mark the CTRL-C-interrupted run as "interrupted" as intended and described in the readme.

Reply:
How should we handle this? First make the merge and than have another look at this topic?

Reply:
Ah, now I read your final comment which answers the question.